### PR TITLE
feat(across): add error for insufficient liquidity

### PR DESCRIPTION
**Pull request by fcescob**
**Motivation**
Add an error for when there is insufficient liquidity on l1 to make a deposit.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: src/components/CoinSelection/CoinSelection.tsx [hunk 1]
```diff
@@ -134,9 +134,14 @@ const CoinSelection = () => {
     ? error.message
     : fees?.isAmountTooLow
     ? "Bridge fee is too high. Try sending a larger amount."
+    : fees?.isLiquidityInsufficient
+    ? `Insufficient liquidity for ${selectedItem?.symbol}.`
     : undefined;
```
Comment (Tulun) [inline src/components/CoinSelection/CoinSelection.tsx:139]: This terniary change is getting a bit messy, maybe it's worth reorganizing it into a function call or something?

File: src/utils/bridge.ts [hunk 2]
```diff
@@ -144,10 +147,25 @@ export async function getLpFee(
...
+    result.pct = calculateRealizedLpFeePct(
+      RATE_MODELS[tokenSymbol],
+      currentUt,
+      nextUt
+    );
     result.total = amount.mul(result.pct).div(parseEther("1"));
   }
+  const liquidityReserves = await bridgePool.liquidReserves();
+  const pendingReserves = await bridgePool.pendingReserves();
+
```
Comment (Tulun) [inline src/utils/bridge.ts:165]: We should wrap these in a try/catch.

**Review by Tulun — APPROVED** [review]
Couple comments, otherwise looks good.

File: src/utils/bridge.ts (hunk 2 above)
Comment (daywiss) [reply src/utils/bridge.ts:165]: this is caught in the chainApi

**Review by daywiss — APPROVED** [review]

**Pull request merged**